content/docs/gke/authentication.md

```diff
@@ -7,6 +7,39 @@ weight = 4
 
 ## In-cluster authentication
 
+Starting from Kubeflow version 0.6, you can consume Kubeflow from custom namespaces (i.e., namespaces other than `kubeflow`).
+The `kubeflow` namespace is intended for running Kubeflow system components while individual jobs and model deployments
```

Comment:
We should also provide info about how this is created and configured as part of profiles.

I think the two key points to get across are

1. KSA to GSA mapping is configured automatically using a profile
1. The GCP service account can be specified in the profile.
1. We should probably link to the page with more info about profiles


Here's a sample spec

```
kubectl -n kubeflow get profiles -o yaml
apiVersion: v1
items:
- apiVersion: kubeflow.org/v1beta1
  kind: Profile
  metadata:
    annotations:
      kubectl.kubernetes.io/last-applied-configuration: |
        {"apiVersion":"kubeflow.org/v1beta1","kind":"Profile","metadata":{"annotations":{},"name":"kubeflow-jlewi"},"spec":{"owner":{"kind":"User","name":"jlewi@google.com"}}}
    creationTimestamp: "2019-10-24T03:55:37Z"
    finalizers:
    - profile-finalizer
    generation: 2
    name: kubeflow-jlewi
    resourceVersion: "5648"
    selfLink: /apis/kubeflow.org/v1beta1/profiles/kubeflow-jlewi
    uid: 232bf931-f612-11e9-8cd6-42010a8e012b
  spec:
    owner:
      kind: User
      name: jlewi@google.com
    plugins:
    - kind: WorkloadIdentity
      spec:
        GcpServiceAccount: kftest-1023-204737-user@jlewi-dev.iam.gserviceaccount.com
    resourcequotaspec: {}
kind: List
```

Reply:
added, ptal